scripts/output_point_cloud.py

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env python
+
+import pyevtk
+import numpy as np
+
+input_filename = 'point_cloud.txt'
+# If output_path is ./point_cloud, the script will output in the current
+# directory a file named point_cloud.vtu
+output_path = './point_cloud'
```

Comment:
Good enough for now.  Please comment that input and output are hard coded at this time and that this of course can be changed.

Reply:
Also please rename `output_point_cloud.py` to something like `export_point_cloud_to_vtk.py`